datalake_api/v0.py

```diff
@@ -224,6 +224,13 @@ def files_get():
                           http_url:
                             type: string
                             description: http url where the file contents
+                          create_time:
```

Comment:
Good to have this in the response, isn't the whole point of the diff so that we can query by `create_time`? Something like `created_before` and `created_after`?

Reply:
No. The point of the diff is that we have create_time in metadata. There's no intention to support new query behavior.

Reply:
Yup, you're completely right